src/tools/rustfmt/src/modules.rs

```diff
@@ -15,6 +16,7 @@ use crate::items::is_mod_decl;
 use crate::parse::parser::{
     Directory, DirectoryOwnership, ModError, ModulePathSuccess, Parser, ParserError,
 };
+
```

Comment:
nit: can you remove this stray newlines.

Reply:
there ya go :)